### ISSUE TITLE
[BUG] H024 should only be reported when type is default

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#78]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3.7
 - djLint Version: 0.5.4

## Issue
When using a script/style with non-default `type`, H024 isnt valid.

## How To Reproduce
```html
<script id="hierarchyTree-template" type="text/kendo-ui-template">
   ...
</script>
```
c.f. https://www.telerik.com/forums/calling-a-text-x-kendo-template-script-type-from-a-javascript-function
